### ISSUE TITLE
flickering on Windows when doing "npm ci"

### ISSUE THREAD

**Issue by flensrocker** [nrwl/nx#11683]
## Current Behavior
When doing a fresh install with `npm ci` on a Windows 11 box there are multiple windows flickering.
I would like to start this in the background and do some other work until this is finished.
The flickering disrupts my work and the input focus is disturbed.

I use PowerShell and Windows Terminal.

## Expected Behavior
No flickering windows.

## Steps to Reproduce
Since I use nx for an Angular mono-repo, I can reproduce it with the following steps:

```
npx create-nx-workspace@latest --preset=angular
# choose org and name etc.
cd org
del .\package-lock.json
del -Recurse .\node_modules
npm install
npm ci
```

In this small example there aren't so much flickering, but there are some.
In my mono-repo with multiple projects there are way more.

### Environment

```
❯ nx report

 >  NX   Report complete - copy this into the issue template

   Node : 16.16.0
   OS   : win32 x64
   npm  : 8.18.0

   nx : 14.5.8
   @nrwl/angular : 14.5.8
   @nrwl/cypress : 14.5.8
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.5.8
   @nrwl/eslint-plugin-nx : 14.5.8
   @nrwl/express : Not Found
   @nrwl/jest : 14.5.8
   @nrwl/js : 14.5.8
   @nrwl/linter : 14.5.8
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : Not Found
   @nrwl/nx-plugin : Not Found
   @nrwl/react : Not Found
   @nrwl/react-native : Not Found
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.5.8
   @nrwl/web : 14.5.8
   @nrwl/workspace : 14.5.8
   typescript : 4.7.4
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
```

**Comment by flensrocker** [comment]
If there are some diagnostics I can run to identify which commands cause the flickering, please tell me.

Thank you!

**Comment by igodorogea** [comment]
The flickering is actually a series of command windows(cmd.exe).
For me it happens on various operations, not just `npm ci`.
Here is an example for branch switch while running `nx serve`

https://user-images.githubusercontent.com/4079076/189354197-e673c3ab-1576-405a-875a-a78404f0a8d5.mp4

**Comment by flensrocker** [comment]
https://github.com/nrwl/nx/pull/7492 seems like a fix to this problem. Maybe there are some other places where it should be applied? Or this fix doesn't work for everyone?

**Comment by FrozenPandaz** [comment]
This should be fixed by https://github.com/nrwl/nx/pull/14965

Can you please try again with the latest version?

**Comment by thomas-ef** [comment]
> This should be fixed by #14965
> 
> Can you please try again with the latest version?

Updated to 15.8.1 and seems to have fixed the issue for me.

What is weird is that the PR you mentioned was released in 15.7.0 but any 15.7.x version for me still made the cmd windows flicker.

**Comment by flensrocker** [comment]
Works for me with nx 15.8.3
